tweepy/user.py

```diff
@@ -63,6 +63,8 @@ class User(HashableID, DataMapping):
         The URL specified in the user's profile, if present.
     verified : bool | None
         Indicates if this user is a verified Twitter User.
+    verified_type : "blue" | "business" | "government" | "none"
+        Indicates the type of verification for the Twitter account.
```

Comment:
Since this field isn't documented for the User object yet, I think it'd be best to copy from the changelog:
```suggestion
        Indicates the type of verification a user account has (blue, business,
        government or none).
```